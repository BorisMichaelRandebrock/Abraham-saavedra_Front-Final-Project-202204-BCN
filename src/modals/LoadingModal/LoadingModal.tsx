import LoadingModalStyles from "./LoadingModalStyles";

const LoadingModal = (): JSX.Element => {
  return (
    <LoadingModalStyles>
      <section className="loading-modal"></section>
    </LoadingModalStyles>
  );
};

export default LoadingModal;
