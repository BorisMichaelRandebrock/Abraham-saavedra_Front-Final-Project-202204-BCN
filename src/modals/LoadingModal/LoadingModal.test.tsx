import reactTestRenderer from "react-test-renderer";
import LoadingModal from "./LoadingModal";

describe("Given a AlertModal componet", () => {
  describe("When it's rendered", () => {
    test("Then it should show always the same as snapshot", () => {
      const modal = reactTestRenderer.create(<LoadingModal />);

      expect(modal).toMatchSnapshot();
    });
  });
});
