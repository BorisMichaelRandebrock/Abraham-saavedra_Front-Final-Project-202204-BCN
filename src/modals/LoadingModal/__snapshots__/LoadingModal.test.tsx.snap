// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a AlertModal componet When it's rendered Then it should show always the same as snapshot 1`] = `
<div
  className="sc-bczRLJ eYbbpD"
>
  <section
    className="loading-modal"
  />
</div>
`;
